htlc/README.md

```diff
@@ -0,0 +1,58 @@
+# HTLC dApp Template
+
+## Hash-Time-Lock-Contract Example in Typescript using PyTeal
+
+There exists a descriptive example explaining how HTLC contracts work with Algo-Builder. It can be found [here](https://github.com/scale-it/algo-builder/tree/master/examples/htlc-pyteal-ts).
+
+Read [here](https://en.bitcoin.it/wiki/Hash_Time_Locked_Contracts) for more information about the HTLC pattern.
+
+Files:
+
+* `assets/htlc.py` : It is the HTLC. SHA256 function is used for hashing. <br />
+        secret value : `sample secret key` hashed with sha256 will be used in this code.
```

Comment:
So this is the correct secret value here, Right? If i try to use another secret value then tx would fail. Can we add this as a **NOTE**?

Reply:
Sure.